docker/README.md

```diff
@@ -1,63 +1,306 @@
-# Docker for itwêwina
+# Docker for morphodict
 
-itwêwina.altlab.app is deployed on a single host with docker-compose.
+The morphodict dictionaries are deployed on a single host with docker-compose.
 
 Here's how it works!
 
-## Overview
+## Quick summary
 
-This `docker-compose.yml` is intended to run on production.
+This `docker-compose.yml` is intended to run on production. Once set up:
+
+    $ ssh itw.altlab.dev
+    $ sudo -i -u morphodict
+    $ cd ~/morphodict/docker
 
     # Runs production by default:
-    docker-compose up
+    docker-compose up -d
 
-The `itwewina` container will expose **port 8001** to the machine. See
-the [application registry]!
+The containers will expose **ports like 8010** to the machine. See
+the [application registry] and [docker/helper/settings.py].
 
 [application registry]: https://github.com/UAlbertaALTLab/deploy.altlab.dev/blob/master/docs/application-registry.tsv
+[docker/helper/settings.py]: https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/blob/main/docker/helpers/settings.py
```

Comment:
This link returns a 404?

Reply:
Yeah, the link is to `docker/helpers/settings.py` on the main branch. It will work once the PR is merged. Until then, you can try https://github.com/andrewdotn/cree-intelligent-dictionary/blob/deploy/docker/helpers/settings.py

Reply:
Ah okay, good good